MusicTool/MusicTool/ClientApp/src/components/Cannon.js

```diff
@@ -23,7 +23,7 @@ const CANNON_SHAPE = [{ x: -20, y: 20 }, { x: 40, y: 0 }, { x: -20, y: -20 }, {
      * @param {any} marbleSize default 20
      * @param {any} marbleCollisionFilter default is all
      */
-     constructor(objectNumber,pos, angle = 0, power = 20, fireLayer = -1, marbleColor = "rand", marbleSize = 20, marbleCollisionFilter = { group: -1, category: 0xFFFFFFFF, mask: 0xFFFFFFFF }, shape = CANNON_SHAPE, collisionFilter = { group: 0, category: 0, mask: 0 }, image = null) {
+     constructor(objectNumber,pos, angle = 0, power = 20, fireLayer = -1, marbleColor = "rand", marbleSize = 20, marbleCollisionFilter = { group: 0, category: 0, mask: 0 }, shape = CANNON_SHAPE, collisionFilter = { group: 0, category: 0, mask: 0 }, image = null) {
```

Comment:
The collision filter needs to be how it was previously because this way will result in new marbles never colliding with any objects created using the default cannon